### DISCUSSION TITLE
Visibility in Home Assistant

### DISCUSSION THREAD

**Discussion by Gugulator** [mercenaruss/zigstar_gateways#21] [Q&A] [answered]
Hello.
I configured only serial port number & speed, and zigbee2MQTT connects to the gateway and receives data from all zigbee devices.
Could you please describe step by step what i need to do to see the status of device in Home Assistant?
I need only get information about new firmware version update available and device temperature.

**Comment by mercenaruss** [comment, marked as answer]
Hi,
you browse to your gateway ip and setup MQTT ,restart and then will get all information about gateway status.

**Reply by Gugulator** [reply]
Yes. I saw this section. But what exactly auto discovery option do? Is it necessary enable it?
Should I change topic or leave it blank?

**Reply by mercenaruss** [reply]
Audiscovery will show all entities available on gateway like temperature,bsl,reset switches and so on. Should be enabled.
You can leave the topic,this doesn't change nothing.

**Reply by Gugulator** [reply]
After enabling mqtt there's system log in Home Assistant

Logger: homeassistant.components.mqtt.discovery
Source: components/mqtt/discovery.py:113
Integration: MQTT ([documentation](https://www.home-assistant.io/integrations/mqtt), [issues](https://github.com/home-assistant/home-assistant/issues?q=is%3Aissue+is%3Aopen+label%3A%22integration%3A+mqtt%22))
First occurred: 22:38:08 (10 occurrences)
Last logged: 22:38:08

Received message on illegal discovery topic 'homeassistant/sensor//uptime/config'. The topic contains not allowed characters. For more information see https://www.home-assistant.io/docs/mqtt/discovery/#discovery-topic
Received message on illegal discovery topic 'homeassistant/sensor//ip/config'. The topic contains not allowed characters. For more information see https://www.home-assistant.io/docs/mqtt/discovery/#discovery-topic
Received message on illegal discovery topic 'homeassistant/sensor//temperature/config'. The topic contains not allowed characters. For more information see https://www.home-assistant.io/docs/mqtt/discovery/#discovery-topic
Received message on illegal discovery topic 'homeassistant/sensor//hostname/config'. The topic contains not allowed characters. For more information see https://www.home-assistant.io/docs/mqtt/discovery/#discovery-topic
Received message on illegal discovery topic 'homeassistant/sensor//connections/config'. The topic contains not allowed characters. For more information see https://www.home-assistant.io/docs/mqtt/discovery/#discovery-topic

**Reply by mercenaruss** [reply]
strange,you are on last firmare.
If NO,then update using ota menu.

**Reply by Gugulator** [reply]
using latest software in all configuration

firmware installed - 0.6.8
zigbee2mqtt - 1.28.0-1
mosquitto broker - 6.1.3
Home Assistant - 2022.10.0
Supervisor - 2022.09.1
Operating System - 9.0
